docs/pages/experiments/joy/components.tsx

```diff
@@ -0,0 +1,253 @@
+import * as React from 'react';
```

Comment:
This file is an experiment page (save to be merged, won't deploy to production)

Reply:
We could also consider adding `fixture` and then see in the future if we break something

Reply:
Added!